hardnet1/main.tf

```diff
@@ -69,10 +99,49 @@ data "template_cloudinit_config" "node" {
 locals {
   node_cloudinit_config = {
     "0" = "${data.template_cloudinit_config.node.0.rendered}"
-    "1" = "${data.template_cloudinit_config.node.1.rendered}"
   }
 }
 
+
+resource "google_compute_instance" "orchestrator" {
```

Comment:
I don't think that orchestrator will need 4 CPU, 16 GB of RAM, 160 GB of SSD.

Reply:
Let's make it `n1-standard-1` and `pd-standard` disk.

Reply:
✅